scripts/docker-compose/imports/objects-api/init/init.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/bash
 
-c=0
-while ! curl http://objecten-api.local:8000
-do
-    c=$((c+1))
-    if [ "$c" = "12" ]; then
-        break
-    fi
-    sleep 10
-done
+set -e
+
+# Wait here a bit for the Objects API database to be initialized (by Objecten).
+# In the end better to add a health check to the Objects API service in Docker Compose but
+# that is not trivial without curl and wget.
+sleep 10
```

Comment:
We already have non-curl healthcheck for Grafana, so we can use the same for objecten-api:

```
    healthcheck:
      test: "bash -c 'printf \"GET / HTTP/1.1\n\n\" > /dev/tcp/127.0.0.1/8000; exit $?;'"
      interval: 20s
      timeout: 10s
      retries: 3
      start_period: 30s
```

Reply:
Or we can keep the above loop with `bash -c 'printf \"GET / HTTP/1.1\n\n\" > /dev/tcp/127.0.0.1/8000; exit $?;'` instead of `curl`

Reply:
ah nice one. Overlooked that. Will have a try!

Reply:
Added a health check according to this Grafana one and it works fine. Thanks!

Reply:
I also updated some other health checks because I think the settings were a bit weird in hindsight.